tests/e2e/e2e_test.go

```diff
@@ -191,13 +192,13 @@ func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
 	var (
 		denom0             string  = "uion"
 		denom1             string  = "uosmo"
-		tickSpacing        uint64  = 1
-		exponentAtPriceOne int64   = -1
+		tickSpacing        uint64  = 100
+		exponentAtPriceOne int64   = -6
```

Comment:
We don't remove exponentAtPriceOne here, as we still want to ensure pools are being created with this value